CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+### 1.0.1
```

Comment:
I suggest making this a new major release as support for older PHP versions is now fully dropped. So a 2.0.0 release is what I would expect here.

Reply:
2.1.0 is already released. Should i write 3.0 ?

Reply:
Yes, I noticed this later. Going for 3.0.0 seems the logical release :+1: